androidApp/src/main/java/com/pulselink/ui/screens/SmsThreadScreen.kt

```
@@ -978,3 +1038,40 @@ private fun LineStatusDot(
+    )
+    resolved.forEach { resolveInfo ->
+        context.grantUriPermission(
+            resolveInfo.activityInfo.packageName,
+            uri,
+            Intent.FLAG_GRANT_READ_URI_PERMISSION
+        )
+    }
+
+    try {
+        context.startActivity(Intent.createChooser(intent, "Send attachment via SMS/MMS"))
+    } catch (e: ActivityNotFoundException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)